Update get_certificate.py

jimwitte opened the pull request:
##### SUMMARY
provide example to calculate number of days until cert expires from get_certificate result.


##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### SUMMARY
<!--- Describe the change below, including rationale and design decisions -->
provide example to calculate number of days until cert expires from get_certificate result.

<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Docs Pull Request

##### COMPONENT NAME
<!--- Write the short name of the module, plugin, task or feature below -->

##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here -->
Calculating how long before a cert expires would seem to be a common task using the output of this module, but took me a while to figure out how to work with the datetime format produced by the module, so wanted to share a solution.

<!--- A step-by-step reproduction of the problem is helpful if there is no related issue -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```paste below

```

jimwitte committed:
Update get_certificate.py

##### SUMMARY
provide example to calculate number of days until cert expires from get_certificate result.


##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

acozine commented:
Thanks @jimwitte for contributing to the Ansible documentation. Your PR currently has two small flaws standing between it and a clean CI run:

```
lib/ansible/modules/crypto/get_certificate.py:110:1: W293 blank line contains whitespace
lib/ansible/modules/crypto/get_certificate.py:115:161: E501 line too long (165 > 160 characters)
```

Can you fix those two issues?

felixfontein commented on lib/ansible/modules/crypto/get_certificate.py:
```diff
@@ -107,6 +107,12 @@
   run_once: true
   register: cert
+  
+- name: how many days until cert expires
```
```suggestion
- name: How many days until cert expires
```

jimwitte committed:
Update lib/ansible/modules/crypto/get_certificate.py

Co-Authored-By: Felix Fontein <felix@fontein.de>

jimwitte committed:
Update get_certificate.py

changed variable name to match "register:cert" from example above.
shortened line to 147 chars
removed whitespace on line 110

acozine commented:
Thanks @jimwitte for the example and @felixfontein for the review.

felixfontein reviewed (approved):
shipit

felixfontein commented:
@jwitte @acozine thanks for your work on this! I've created a backport to stable-2.8: #56758
